packages/patternfly-4/react-docs/gatsby-browser.js

```diff
@@ -1,6 +1,4 @@
 // Import global CSS files here. Have no remorse.
-import '@patternfly/patternfly/patternfly.css';
 // https://www.gatsbyjs.org/docs/creating-global-styles
-import './static/base.css';
-import '@patternfly/patternfly/components/Table/table.css';
-import '@patternfly/patternfly/components/Table/table-grid.css';
```

Comment:
Are these not needed?

Reply:
Dana added `@patternfly/patternfly/patternfly.css`, so `@patternfly/patternfly/components/Table/table.css` and `@patternfly/patternfly/components/Table/table-grid.css` aren't needed anymore.

Reply:
k, thanks